### PR TITLE
Update Operator SDK to v1.22.2

### PR DESCRIPTION
**Pull request by Red-GV**
This PR updated operator-sdk to v1.22.2, bingo to v.0.6.0, and golangci-lint to v1.47.2. This also fixes the lint errors that occur from it.

/cc @aminesnow 
/assign @periklis

### CODE REVIEW

File: internal/elasticsearch/certificates.go [hunk 1]
```diff
@@ -557,8 +556,15 @@ func ensureSessionSecret(secretLength int, runes []rune, session *sessionSecret)
...
+	bigRuneCount := big.NewInt(int64(len(runes)))
+
 	for i := range b {
-		b[i] = runes[mathRand.Intn(len(runes))]
+		bigIndex, err := rand.Int(rand.Reader, bigRuneCount)
+		if err != nil {
+			return err
+		}
+
+		b[i] = runes[int(bigIndex.Int64())]
```
Comment (periklis) [inline internal/elasticsearch/certificates.go:567]: What does this change have in common with updating the sdk? Is this more a Golang 1.18 thing?

Comment (Red-GV) [reply internal/elasticsearch/certificates.go:567]: This is a golangci lint error for golang 1.18 as far aas I'm aware. I can break this into two PRs if needed.

Comment (periklis) [reply internal/elasticsearch/certificates.go:567]: No it's ok here

**Review by periklis — APPROVED** [review]

**Pull request merged**